src/status_im2/contexts/chat/lightbox/top_view.cljs

```diff
@@ -39,14 +42,33 @@
         (anim/animate top-view-width screen-width)
         (anim/animate top-view-bg colors/neutral-100-opa-0)))))
 
+(defn drawer
+  [uri]
+  [quo/action-drawer
+   [[{:icon                :i/save
+      :accessibility-label :save-image
+      :label               (i18n/label :t/save-image-library)
+      :on-press            (fn []
+                             (rf/dispatch [:hide-bottom-sheet])
+                             (images/save-image-to-gallery
```

Comment:
Hey @ilmotta,
I am wondering what would the advantage of dispatching an event instead of calling the function directly, as we are not doing any updates to app DB